react-rpg/src/Components/Container/Container.jsx

```diff
@@ -0,0 +1,12 @@
+import './container.css';
+
+const Container = ( props ) => {
```

Comment:
This component seems a little unnecessary for now, lets remove it and just keep the class